proto/forch_configuration.proto

```diff
@@ -150,6 +150,9 @@ message OrchestrationConfig {
 
     // DUT will return to operational state if no DeviceReport is received within this timeout.
     int32 sequester_timeout_sec = 6;
+
+    // List of device macs that will be sequestered for testing
+    repeated string allowed_list = 7;
```

Comment:
"allowed" doesn't seem the right word here... I'd call this something like "target_macs" or "enabled_macs" or "sequestered_macs" -- the generalized concept would be to properties that could be applied to a list of device macs?